phoenix-core/src/main/java/org/apache/phoenix/coprocessor/UngroupedAggregateRegionObserver.java

```diff
@@ -165,6 +165,9 @@ public RegionScanner preScannerOpen(ObserverContext<RegionCoprocessorEnvironment
             throws IOException {
         s = super.preScannerOpen(e, scan, s);
         if (ScanUtil.isAnalyzeTable(scan)) {
+            if (!ScanUtil.isLocalIndex(scan)) {
+                scan.getFamilyMap().clear();
+            }
```

Comment:
Can we file a JIRA for this? MetaDataClient.updateStatisticsInternal() would just need to project the correct column family here.